docs/core-concepts/index.md

```diff
@@ -0,0 +1,32 @@
+# Core Concepts
+
+These are some of the core concepts that you should be familiar with before going over the [Getting Started Guide](get-started/index.md).
+
+
+### **Git Source**
```

Comment:
This should really be `##`. HTML `h*` tags define nesting, and there can thus not be a `h3` without a higher level `h2`.